app/app.go

```diff
@@ -100,6 +101,10 @@ func Run(ctx context.Context, kubeConfig rest.Config, cfg *Config) error {
 			panic(err)
 		}
 
+		if err := whitelist.Load(management.Management.Settings(""), management.Management.NodeDrivers("")); err != nil {
```

Comment:
Can we just remove this?  This is only called on the master in a HA setup.  So it's not generically ran.